.ci/ansible/tasks/install_ssh_keys.yml

```
@@ -1,5 +1,12 @@
 ---
-- name: Install SSH keys
+- name: Install SSH keys (Linux)
   become: false
   shell: |
     /home/{{ansible_user}}/e2e-testing/.ci/scripts/import-ssh-keys.sh
+  when: ansible_os_family != 'Darwin'
+
+- name: Install SSH keys (MacOS)
+  become: false
+  shell: |
+    /Users/{{ansible_user}}/e2e-testing/.ci/scripts/import-ssh-keys.sh
+  when: ansible_os_family == 'Darwin'
```

I think we can simplify the task to be cross-platform using the var added in this PR (Windows not supported yet)
```suggestion
- name: Install SSH keys (Linux and MacOS)
  become: false
  shell: |
    /{{ ansible_home_path }}/{{ ansible_user }}/e2e-testing/.ci/scripts/import-ssh-keys.sh
```